### ISSUE TITLE
pokemon sword and shield raid error

### ISSUE THREAD

**Issue by kik4186** [spacemeowx2/ldn_mitm#26]
when on idnmitm for pokemon shield when in raids no more than 1 person can join

**Comment by takashi1kun** [comment]
yeah it seems the max are 2 ppl raids, the creator and one guest

**Comment by slopyz** [comment]
> yeah it seems the max are 2 ppl raids, the creator and one guest

Yeah I agree with that. May you fix this bug please ? Can you explain if you find something ? Thanks in advance

**Comment by spacemeowx2** [comment]
is it still a issue? since pokemon sword and shield has lan mode
